pkg/restore/restore.go

```diff
@@ -1062,13 +1062,13 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 					ctx.log.Infof("ServiceAccount %s successfully updated", kube.NamespaceAndName(obj))
 				}
 			default:
-				e := errors.Errorf("not restored: %s and is different from backed up version.", restoreErr)
+				e := errors.Errorf("error restoring %s: not possible since it already exists in the cluster but is different from the backed up version.", restoreErr)
```

Comment:
Maybe splitting it into two sentences would help - something like "Couldn't restore ConfigMap foo/bar because it already exists in the cluster. Warning: the in-cluster version is different than the backed-up version".

Reply:
I like splitting them into 2 sentences, too.